Desktop/Prework/Allegro/allegro/src/components/Loader/Loader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import LinearProgress from '@material-ui/core/LinearProgress';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        width: '100%',
+        '& > * + *': {
```

Comment:
We should be very careful when using `*`, because it makes browser suffer. If we can we should replace it with more specific selectors